### ISSUE TITLE
Cron job format 

### ISSUE THREAD

**Issue by MahmoudSamy511** [Martin-Laclaustra/CronAlarms#28]
Hi there, 
Recently i tried to use a format like this : "* 15-18 * * *" which will call function for  every minute past every hour from 15 through 18. when I tried to do this the Cron.Create() returns 255 and it does not schedule this task so what should I do ? 
or the library supports only some of from format not all formats .

**Comment by Martin-Laclaustra** [comment]
According to https://github.com/staticlibs/ccronexpr , the code that this library wraps, it should work.
Sorry, currently I can not do tests or fix the code. My advice is to do elemental tests with the ccronexpr.h and ccronexpr.c files to try to pinpoint your problem.

**Comment by MahmoudSamy511** [comment]
I have tried to create a job with this format  "* 15-18 * * *"  as I said before but the function create returns 255.
so that indicates the job has not been created. I tried to schedule a job in this format : */120 * * * * * it should call a function every 120 seconds but it calls it every 60 seconds so my question about is this library supports all corn formats? and if it is, what is the problem with these two formats? And what is the matter if I tested the format with the ccronexpr.h and ccronexpr.c files and if it works in the CronAlarm library?
